### ISSUE TITLE
RaspiBlitz waiting for start/ LND waiting to sync

### ISSUE THREAD

**Issue by Sixpaths7** [raspiblitz/raspiblitz#4021]
Hello Guys!

Im a new and am downloading the blockchain to my rapsiblitz. At around 20% it stopped and had the standard screen of:
Blockchain waiting for start
LND waiting to sync

I tried rebooting the Blitz but nothing has happened. I can still access the menu.
Can anyone help me please? Is a bug report helpful?

**Comment by rootzoll** [comment]
Yes please post the results when you type `debug` on the commandline.
